client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -1177,14 +1177,14 @@ Params withWaitForEvents(Priority waitForEvents) {
      */
     static XContentType enforceSameContentType(IndexRequest indexRequest, @Nullable XContentType xContentType) {
```

Comment:
this is to assert that during bulk api all requests have the same contentype. Used by rest HLRC